discover/router.go

```diff
@@ -0,0 +1,94 @@
+package discover
```

Comment:
comment provided offline to avoid re-building everything that already exists

Reply:
ticker set to updater